docker/build.sh

```diff
@@ -114,9 +124,10 @@ function upsearch () {
 # reasonable defaults if you run it outside of Jenkins.
 WORKSPACE="${WORKSPACE:-${SCRIPT_DIR}/../}"
 BUILD_TAG="${BUILD_TAG:-tvm}"
+DOCKER_IMAGE_TAG="${DOCKER_IMAGE_TAG:-latest}"
 
 # Determine the docker image name
-DOCKER_IMG_NAME="${BUILD_TAG}.${CONTAINER_TYPE}"
+DOCKER_IMG_NAME="${BUILD_TAG}.${CONTAINER_TYPE}:${DOCKER_IMAGE_TAG}"
```

Comment:
maybe want to do this after sanitizing the DOCKER_IMG_NAME below? or maybe more ideally assemble DOCKER_IMG_SPEC and use that in place of DOCKER_IMG_NAME on the command-line below

Reply:
I think I made this change. Have a look.